packages/react-components/react-drawer/src/components/DrawerOverlay/useDrawerOverlayStyles.styles.ts

```diff
@@ -11,26 +11,26 @@ import {
   useDrawerDurationStyles,
 } from '../../shared/useDrawerBaseStyles.styles';
 
-export const drawerOverlayClassNames: Omit<SlotClassNames<DrawerOverlaySlots>, 'dialog'> = {
+export const drawerOverlayClassNames: SlotClassNames<DrawerOverlaySlots> = {
   root: 'fui-DrawerOverlay',
   backdrop: 'fui-DrawerOverlay__backdrop',
 };
 
 /**
  * Styles for the root slot
  */
-const useDrawerRootStyles = makeStyles({
-  root: {
-    ...drawerDefaultStyles,
-    position: 'fixed',
-    top: 0,
-    bottom: 0,
-    opacity: 0,
-    boxShadow: '0px transparent',
-    transitionProperty: 'transform, box-shadow, opacity',
-    willChange: 'transform, box-shadow, opacity',
-  },
+const useDrawerResetStyles = makeResetStyles({
+  ...drawerDefaultStyles,
+  position: 'fixed',
+  top: 0,
+  bottom: 0,
```

Comment:
Do you need both here `top` and `bottom`?

Reply:
That's needed as Drawer needs to be anchored to top and bottom.